notebooks/00_quick_start/lstur_synthetic.ipynb

```diff
@@ -0,0 +1,316 @@
+{
```

Comment:
<p>could we please change all the <code>import *</code>  to import the actual functions that we are using in the notebook? In the rest of the notebooks we are trying to follow that rule</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1080/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>